kube-runtime/src/controller/mod.rs

```diff
@@ -363,12 +446,17 @@ where
     pub fn owns<Child: Clone + Resource + DeserializeOwned + Debug + Send + 'static>(
         mut self,
         api: Api<Child>,
+        dyntype: Child::DynamicType,
         lp: ListParams,
     ) -> Self
     where
-        Child::DynamicType: Debug + Eq + Hash,
+        Child::DynamicType: Debug + Eq + Hash + Clone,
     {
-        let child_watcher = trigger_owners(try_flatten_touched(watcher(api, lp)), self.dyntype.clone());
+        let child_watcher = trigger_owners(
+            try_flatten_touched(watcher(api, lp)),
+            self.dyntype.clone(),
+            dyntype,
+        );
```

Comment:
actually question: why does `dyntype` need to be explicitly passed in now?

Reply:
nevermind, i see it's for getting info for the reason. damnit, i thought we had that info before. that's kind of annoying. i'm slightly sad that i moved dyntype out of `Api` now.